gov_uk_dashboards/components/plotly/header.py

```diff
@@ -11,46 +11,48 @@ def header(title):
     """
     return html.Header(
         html.Div(
-            html.Div(
-                [
-                    html.Img(
-                        src="assets\\images\\hm-government-logo.png",
-                        style={"width": "62px"},
-                    ),
-                    html.Span(
-                        [
-                            "Department for Levelling Up,",
-                            html.Br(),
-                            "Housing & Communities",
-                        ],
-                        className=" ".join(
-                            [
-                                "govuk-header__link",
-                                "govuk-header__link--service-name",
-                                "govuk-!-padding-left-3",
-                                "govuk-!-font-size-14",
-                            ]
+            [
```

Comment:
There was some conversation about there being duplicated `header.py` files both within this repo and the dashboard repos.  Should this be the case?  Do the downstream repos need to use this `header.py`?